common-libs/common/tsconfig.json

```diff
@@ -3,7 +3,7 @@
     "module": "commonjs",
     "resolveJsonModule": true,
     "esModuleInterop": true,
-    "target": "ES2016",
+    "target": "ES2019",
```

Comment:
(for `Array.prototype.flatMap` support)

Reply:
Does this mean that the final build of the SDK won't be supported in browsers that don't support ES2019? (not that we really care about that, as I see).

Reply:
Some other packages already rely on ES2019.

If we care about old browsers, we should probably use some polyfills (and `flatMap` is trivially implemented using `map` and `flatten`) instead of trying to limit ourselves to outdated APIs.